data-transform.qmd

```diff
@@ -372,6 +377,8 @@ flights |>
 But you can use the same `.before` and `.after` arguments as `mutate()` to choose where to put them:
 
 ```{r}
+#| results: false
```

Comment:
I know we're hiding the code output to save space, but it's sad that code and output can't always be presented, it's much nicer when we can do that.

Reply:
Agreed